rabbitmq-scenario/config/docker-compose.yml

```diff
@@ -0,0 +1,42 @@
+version: '2.1'
+services:
+  rabbitmq-server:
+    image: rabbitmq:latest
+    expose:
+    - "5672"
+    ports:
+      - "5672:5672"
```

Comment:
Remove the mapping ports. Otherwise it will cause the exception that the port is already allocated.